.idea/misc.xml

```diff
@@ -2,7 +2,10 @@
 <project version="4">
   <component name="ExternalStorageConfigurationManager" enabled="true" />
   <component name="JavaScriptSettings">
-    <option name="languageLevel" value="JSX" />
+    <option name="languageLevel" value="ES6" />
```

Comment:
Should we just add `.idea` to gitignore? Because this will get out of hand fast.